soh/include/z64save.h

```diff
@@ -25,6 +25,12 @@ typedef struct {
     /* 0x5C */ s16 gsTokens;
 } Inventory; // size = 0x5E
 
+typedef struct {
+    /*      */ u8 heartPieces;
+    /*      */ u8 heartContainers;
+    /*      */ u8 dungeonKeys[19];
+} Stats;
```

Comment:
i feel like there's got to be a better name than stats but nothing is coming to mind, anyone have any ideas?

Reply:
> Stats because I imagine this is where we will store other stat related things like bonks, time played, damage taken, etc.

Pasting here in case you didn't see other comment thread. I'm open to ideas though

Reply:
yeah, i saw that, that's part of what's making it so hard to think of a good name

it needs to be general enough to account for all of those, but still communicate that it's not including fishing high scores and stuff

Reply:
i think `Stats` is good but a little too generic, maybe `PlayerStats` or `GameplayStats` or something

Reply:
or maybe SohStats (because they aren't vanilla stats)

Reply:
Since GlobalContext is going to be renamed to PlayState at some point, maybe piggyback off of that and call it PlayStats?